manifest.json

```diff
@@ -59,19 +83,76 @@
   },
   "contributions": [
     {
-      "id": "google-cloud-service-endpoint",
+      "id": "google-cloud-endpoint-ui",
+      "displayname": "Google Cloud Platform service endpoint UI",
+      "type": "ms.vss-endpoint.service-endpoint-ui",
+      "targets": [
+        "ms.vss-endpoint.endpoint-ui-catalog"
+      ],
+      "properties": {
+        "file": "main.html"
+      }
+    },
```

Comment:
This contribution is complicated enough that it should be in its own folder with a separate manifest.json.